# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bbdock/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- bbdock-0.2.8/src/Icon.cc.vanilla	2009-02-02 09:54:46.000000000 +0100
+++ bbdock-0.2.8/src/Icon.cc	2009-02-02 09:54:53.000000000 +0100
@@ -29,6 +29,8 @@
 #include <string>
 #include <vector>
 
+#include <stdlib.h>
+
 #include "Icon.hh"
 
 using namespace std;
--- bbdock-0.2.8/src/WindowManager.cc.vanilla	2009-02-02 09:53:45.000000000 +0100
+++ bbdock-0.2.8/src/WindowManager.cc	2009-02-02 09:54:40.000000000 +0100
@@ -33,6 +33,7 @@
 
 #include <iostream>
 #include <string>
+#include <string.h>
 
 #include "WindowManager.hh"
 
